layouts/articles/single.html

```diff
@@ -87,7 +88,7 @@ <h5 class="text-16 text-blockquote">
           console.log(images[2].src);
         </script>
 
-        {{ else }} {{ end }}
+        {{ end }}
       </div>
     </div>
   </section>
```

Comment:
Also further below is another newline I fixed with a different commit - Line 139.